src/main/java/com/github/tomakehurst/wiremock/core/WireMockConfiguration.java

```diff
@@ -142,6 +156,11 @@ public static WireMockConfiguration options() {
     return wireMockConfig();
   }
 
+  public WireMockConfiguration proxyPassThrough(Boolean proxyPassThrough) {
```

Comment:
Can we change this to `boolean`?

Reply:
Won't be required if I won't incorporate above comment.

Reply:
incorporated